src/utilities/qa.js

```diff
@@ -66,7 +54,7 @@ export const createLookupPromises = (client, query, lookupConfigs) => lookupConf
    */
   const actionFunction = subauthority ? subAuthCall : authorityCall
 
-  return client
+  return Swagger({ spec: swaggerSpec }).then(client => client
```

Comment:
Does this cause the swagger to be reloaded every time this is called? Would that be slow?